.github/workflows/ci-minimized.yml

```diff
@@ -0,0 +1,254 @@
+                    docker/sdk cli vendor/bin/console setup:init-db
+                    docker/sdk cli vendor/bin/console data:import --config=data/import/common/minimal.yml
+                    docker/sdk cli vendor/bin/console search:setup:sources
+                    docker/sdk cli vendor/bin/glue glue-api:controller:cache:warm-up
+                    docker/sdk cli vendor/bin/glue api:generate:documentation
+
+            -   name: Publish & Sync
+                run: |
+                    docker/sdk testing console queue:worker:start --stop-when-empty
+
+            -   name: Fixtures
+                run: |
+                    docker/sdk testing codecept fixtures
+
+            -   name: Glue Acceptance CheckoutMinimumWorkflow
+                run: |
+                    docker/sdk testing codecept run -c codeception.api.yml -g CheckoutMinimumWorkflow
+
+            -   name: Yves Acceptance CheckoutMinimumWorkflow
+                run: |
+                    docker/sdk testing codecept run -c codeception.acceptance.yml -g YvesCheckoutMinimumWorkflow
+
+            -   name: Save PHP logs
+                if: failure()
+                run: |
+                    mkdir -p logs
+                    docker logs spryker_ci_backoffice_eu_1 > logs/backoffice_app.log 2>&1
+                    docker logs spryker_ci_glue_eu_1 > logs/glue_app.log 2>&1
+                    docker logs spryker_ci_yves_eu_1 > logs/yves_app.log 2>&1
+                    docker logs spryker_ci_merchant_portal_eu_1 > logs/merchant_portal_app.log 2>&1
+
+            -   name: Upload Failed Screenshots and HTML
+                if: always()
+                uses: actions/upload-artifact@v4
+                with:
+                    name: codeception-failed-artifacts
+                    retention-days: 5 # save for 5 days
+                    path: |
+                        logs/*.log
+                        tests/**/_output/**/*
+                        vendor/spryker/spryker/Bundles/**/tests/_output/*
+
+            -   name: Frontend Assets
+                run: |
+                    SPRYKER_BUILD_HASH=myhash docker/sdk export images
+                    docker run -i --rm -e SPRYKER_NGINX_CGI_HOST_YVES_EU=localhost -e SPRYKER_NGINX_CGI_HOST_BACKOFFICE_EU=localhost -e SPRYKER_NGINX_CGI_HOST_BACKEND_GATEWAY_EU=localhost -e SPRYKER_NGINX_CGI_HOST_BACKEND_API_EU=localhost -e SPRYKER_NGINX_CGI_HOST_GLUE_EU=localhost -e SPRYKER_NGINX_CGI_HOST_CONFIGURATOR_EU=localhost -e ALLOWED_IP=127.0.0.1 spryker_ci_frontend:1.0-frontend nginx -t
+                    docker run -i --rm --entrypoint='' spryker_ci_frontend:1.0-frontend ls -al /data/public/Yves/assets/myhash
+                    docker run -i --rm --entrypoint='' spryker_ci_frontend:1.0-frontend ls -al /data/public/Zed/assets/
+                    docker run -i --rm --entrypoint='' spryker_ci_frontend:1.0-frontend ls -al /data/public/MerchantPortal/assets/
```

Comment:
I still don't see why we need to run Docker build, which installs something inside

<img width="1081" height="1168" alt="Image" src="https://github.com/user-attachments/assets/df343d41-d9d2-4de1-a1da-fa142ba615c8" />

Then run Nginx (frontend image), then check that js/css files are generated inside the container.

We don't run this `suite` project in Cloud, in the production...

During the `docker/sdk up` the assets have to be already there

I suggest instead only:
```
docker/sdk cli ls -alR /data/public/Yves/assets/
docker/sdk cli ls -al /data/public/Zed/assets/
docker/sdk cli ls -al /data/public/MerchantPortal/assets/
```

Reply:
let's check together

I see in `config/install/docker.ci.fe-api.yml`  sections

- build-static
- build-static-production
- build-static-development

They should be executed during `docker/sdk up -t`